django/contrib/auth/middleware.py

```diff
@@ -21,7 +20,6 @@ def process_request(self, request):
             "'django.contrib.sessions.middleware.SessionMiddleware' before "
```

Comment:
Even though `AuthenticationMiddleware` is no longer useful, it needs to remain as a stub and go through a deprecation.